docs/docs/guides/tasks.md

```diff
@@ -199,22 +199,30 @@ page.
 
 ## Context
 
-A Kubernetes context is a set of access parameters
-that contains a Kubernetes cluster, a user, a namespace,
-the application name, workload name, and version.
-For more information, see
-[Configure Access to Multiple Clusters](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
-
-You may need to include context information in the `function` code
-included in the YAML file that defines a
-[KeptnTaskDefinition](../reference/crd-reference/taskdefinition.md)
-resource.
-For an example of how to do this, see the
-[keptn-tasks.yaml](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd/blob/main/simplenode-dev/keptn-tasks.yaml)
-file.
-
-A context environment variable is available via `Deno.env.get("KEPTN_CONTEXT")`.
-It can be used like this:
+In Kubernetes the concept of context refers to the information needed to 
```

Comment:
These two intro paragraphs are rewritten in https://github.com/keptn/lifecycle-toolkit/pull/2839 , which I have approved.  @mowies has a block on it but I think the issue is resolved by the rewrite.  But let's be sure to get that version of this in he revised doc here

Reply:
Yeah I guess this means we need to hold this pr until the other one is merged!

Reply:
Hopefully Mo can approve it this morning and we can merge it.

Reply:
it's approve and waiting for a final check from you @StackScribe :)